### PR TITLE
Track aborted tests in spanner

### PR DESCRIPTION
**Pull request by therealmitchconnors**
aborted tests have been showing up as test flakes, exaggerating the rate of flakes.  This PR adds awareness of aborted tests, as well as tests that are still in progress.

### CODE REVIEW

**Comment by ericvn** [comment]
Will need a `gofmt -s-w policybot/pkg/resultgatherer/types.go` to get the fmt changes required to pass lint.

File: policybot/pkg/resultgatherer/resultgatherer.go [hunk 1]
```diff
@@ -447,39 +488,59 @@ func (trg *TestResultGatherer) GetPostSubmitTestResult(ctx context.Context, test
...
 	}
+	if pj.Status.State == SuccessState || pj.Status.State == FailureState {
+		records, err := trg.getInformationFromCloneFile(ctx, testRun)
+		if err != nil {
+			return nil, err
+		}
 
-	testResult.BaseSha = record.Refs.BaseSha
-	testResult.CloneFailed = record.Failed
+		if len(records) < 1 {
```
Comment (richardwxn) [inline policybot/pkg/resultgatherer/resultgatherer.go:515]: Seems like this part of logic is similar with that in getInformationFromCloneFile, can possibly moved to common function?

**Comment by richardwxn** [comment]
LGTM other than the above comment, approving it now, feel free to remove the do-not-merge label after you address the comment.

**Review by richardwxn — APPROVED** [review]

File: policybot/pkg/resultgatherer/resultgatherer.go (hunk 1 above)
Comment (therealmitchconnors) [reply policybot/pkg/resultgatherer/resultgatherer.go:515]: Chatted offline.  Difficult to consolidate this logic due to minor differences.

**Pull request merged**